src/components/Layout/Layout.styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+
+//components
+import { NavLink } from 'react-router-dom';
+
+//icon
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+//interface
+import { ShowingProps } from '@interface';
+
+//theme
+import theme from '@/theme/theme';
+
+export const LayoutContainer = styled.div`
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 10;
+`;
+
+export const HeaderWrapper = styled.header``;
+
+export const HeaderImg = styled.div`
+    position: relative;
+    height: 100px;
+    background-image: url('../assets/img/header.png');
```

Comment:
Nie używaj background-img, użyj zwykłego img z src. Bo to wpływa na SEO. Tak, wiem, w React nie istnieje SEO jako takie, ale lepiej mieć dobre praktyki.